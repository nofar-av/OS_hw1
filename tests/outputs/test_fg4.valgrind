==27100== Memcheck, a memory error detector
==27100== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==27100== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==27100== Command: /home/omer/OS_wet1/smash
==27100== Parent PID: 27096
==27100== 
==27100== 
==27100== FILE DESCRIPTORS: 4 open at exit.
==27100== Open file descriptor 3: /home/omer/OS_wet1/tests/outputs/test_fg4.valgrind
==27100==    <inherited from parent>
==27100== 
==27100== Open file descriptor 2: /home/omer/OS_wet1/tests/outputs/test_fg4.err
==27100==    <inherited from parent>
==27100== 
==27100== Open file descriptor 1: /home/omer/OS_wet1/tests/outputs/test_fg4.out
==27100==    <inherited from parent>
==27100== 
==27100== Open file descriptor 0:
==27100==    <inherited from parent>
==27100== 
==27100== 
==27100== HEAP SUMMARY:
==27100==     in use at exit: 0 bytes in 0 blocks
==27100==   total heap usage: 113 allocs, 113 frees, 86,676 bytes allocated
==27100== 
==27100== All heap blocks were freed -- no leaks are possible
==27100== 
==27100== For counts of detected and suppressed errors, rerun with: -v
==27100== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
