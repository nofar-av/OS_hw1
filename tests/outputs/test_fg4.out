smash> submitted by Eran
smash> smash> smash> smash> smash> smash> sleep 200& : 28428
smash: got ctrl-Z
smash: process 28428 was stopped
smash> [1] sleep 100& : 28427 0 secs
[2] sleep 200& : 28428 0 secs (stopped)
smash> sleep 100& : 28427
smash: got ctrl-C
smash: process 28427 was killed
smash> [2] sleep 200& : 28428 0 secs (stopped)
smash> smash> sleep 300& : 28442
smash: got ctrl-Z
smash: process 28442 was stopped
smash> [2] sleep 200& : 28428 0 secs (stopped)
[3] sleep 300& : 28442 0 secs (stopped)
smash> smash: sending SIGKILL signal to 2 jobs:
28428: sleep 200&
28442: sleep 300&
